docs/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - `[Modal]` Added a new SohoModalModule that would work almost similar to SohoModalDialogModule but fixes memory leak issues. `NBCP` ([#639](https://github.com/infor-design/enterprise-ng/pull/639))
 - `[Datagrid]` Made the last two options on exportToExcel optional. Note that passing in a third option is incorrect and will give you no headers in the exported file, so generally shouldnt be used. `TJM` ([#654](https://github.com/infor-design/enterprise-ng/pull/654))
 - `[Dropdown]` Added keydown output EventEmitter. `NBCP` ([#658](https://github.com/infor-design/enterprise-ng/pull/658))
+- `[Modal]` Merged SohoModalModule into SohoModalDialogModule keeping current API but fixing the memory leaks. `BTHH` ([#663](https://github.com/infor-design/enterprise-ng/pull/663))
```

Comment:
Can you add a section for 7.0 and put this here. This branch is moving towards 7.0
If we need this fixed in 6.3.0 i can make a branch for that. Presently that would be off 6.2.x 

Or does this negate the need for a 7.x?

Reply:
I think until we have an `ng9` PR ready we should hold off on a `7.x`. @bthharper where are you on that one Theo?

Reply:
Sure it can be whenever but the master branch has a breaking change in it that will require a 7.x release? Unless that was fixed?

Reply:
@pwpatton  - not that far down the road, was waiting for the release to look at it.  I do have an IVY branch that I was working on.    

@tmcconechy - what's the breaking change?

Reply:
I thought the previous https://github.com/infor-design/enterprise-ng/pull/639 PR. But if we are happy to release that into a 6.3 maybe we are OK?

Reply:
Should we just merge everything in master and make it a 6.3 next even though there is a minor breaking change?

Reply:
on discussing with Phillip lets keep master going to 6.3.x for now. Despite that change. 
So all this can go in master.